liquidprompt

```diff
@@ -929,6 +935,27 @@ __lp_theme_zsh_complete() {
     _describe 'theme' lp_theme_list
 }
 
+_lp_hash_color() {
+    # compute the hash of the argument and color it with the corresponding color in:
+    # (green, yellow, blue, purple, cyan)
+    # Red (would be index 1) is not used here, as it should be reserved for alerts.
+    local str=${1-}
+    local cksum="$(printf '%s' "$str" | cksum)"
+    local hash="${cksum%%[$' \t']*}"
+    __lp_foreground_color "$(( 2 + hash % 5 ))"
```

Comment:
By skipping red, you prevent this from being used for `__lp_hostname_hash()` (since we can't change that now without changing all hostname color mappings).

I get your logic, but maybe there is a way to make it work yet?

```suggestion
    __lp_foreground_color "$(( 2 + (hash - 1) % 6 ))"
```
This would keep previous colors the same, but change red to white. Hm, white probably isn't the best choice. Maybe it can't work, but I'll leave my thoughts here anyway.

Reply:
I thought a bit about that, but red seems less a problem for the hostname, as I can't think of a "dangerous" host.
However, I can think of "dangerous" development environment states, that I would not want to be mistaken with other states.

So at the end I just let the hostname code as is.